### PR TITLE
Add jq and httpie to dependencies and remove "beta2" from urls.

### PR DESCRIPTION
**Pull request by stevana**

### CODE REVIEW

File: templates/platform/kafka.tmpl.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ data:
   KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: "1"
   KAFKA_ZOOKEEPER_CONNECT: "{{ .zookeeper_host }}:2181"
 ---
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```
Comment (taheris) [inline templates/platform/kafka.tmpl.yaml]: Is there a reason to change this? It seems `apps/v1` was only added in 1.9 and we want to support 1.8 for now at least.

Comment (stevana) [reply templates/platform/kafka.tmpl.yaml]: I got 1.9 and the `beta2` thing doesn't work for me. Is it possible to support both somehow?

Comment (taheris) [reply templates/platform/kafka.tmpl.yaml]: What error are you getting? According to [the docs](https://kubernetes.io/docs/reference/workloads-18-19/) `beta2` should be supported in 1.9, and auto-converted in 1.10+

Comment (stevana) [reply templates/platform/kafka.tmpl.yaml]: Can't reproduce it now. I put the "beta2" back.

Comment (stevana) [reply templates/platform/kafka.tmpl.yaml]: By the way, the error was:

```service "zookeeper" created
Error from server (BadRequest): error when creating ".platform.yaml": the API version in the data (apps/v1beta2) does not 
match the expected API version (apps/v1)
Error from server (BadRequest): error when creating ".platform.yaml": the API version in the data (apps/v1beta2) does not 
match the expected API version (apps/v1)
Error from server (BadRequest): error when creating ".platform.yaml": the API version in the data (apps/v1beta2) does not 
match the expected API version (apps/v1)
Makefile:56: recipe for target 'start-platform' failed
```

**Pull request merged**